openedx/core/djangoapps/user_api/urls.py

```diff
@@ -4,7 +4,7 @@
 
 from .accounts.views import AccountView
 from .preferences.views import PreferencesView, PreferencesDetailView
-
+from ..profile_images.views import ProfileImageView
 from django.conf.urls import patterns, url
```

Comment:
I saw that, but didn't realize that ..imports were considered non-local.  Should dotted-imports be sorted by descending number of leading dots?